certbot-apache/certbot_apache/configurator.py

```diff
@@ -948,6 +954,12 @@ def _add_name_vhost_if_necessary(self, vhost):
         if need_to_save:
             self.save()
 
+    def _unescape(self, fp):
+        return fp.replace("\\", "")
+
+    def _escape(self, fp):
+        return fp.replace(",", "\\,")
```

Comment:
It may be worth leaving a comment here that this is (partially) equivalent to aug_escape_name, which is in 1.4.0 but not yet added to python-augeas. In future it could be swapped out, which'd make it reliable and correct for some other characters (such as `[]`).